core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -41,7 +41,7 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
-    $scope.myExplorationsView = 'list';
+    $scope.myExplorationsView = 'card';
```

Comment:
Yeah, I agree. But figured out that the changes are very minor, so accommodated those in this PR only!